Solaris/MoonPhase.php

```diff
@@ -44,6 +44,9 @@ class MoonPhase
 	/** @var array quarters */
 	protected $quarters = false;
 
+	/** @var array degrees */
+	protected $moondegree;
```

Comment:
I'd suggest we call this `age_in_degrees` for clarity. The comment above needs to use the same name.